HW04/ivan_klimov/spec/student_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require_relative '../lib/person'
+require_relative '../lib/mentor'
+require_relative '../lib/student'
+require_relative '../lib/homework'
+require_relative '../lib/notification'
+
+RSpec.describe Student do
+  let(:mentor) { Mentor.new(name: 'Olga', surname: 'Urban') }
+  let(:student) { described_class.new(name: 'Ked', surname: '500') }
+  let(:homework) { Homework.new(title: 'task1', description: 'Solve 5/6', student: student) }
+
+  describe '#mark_as_read' do
+    it 'clean student notifications ' do
+      student.mark_as_read!
+      expect(student.notifications).to eq student.notifications
```

Comment:
try to use `expect {}.to change {}.from().to()` , please

Reply:
thanks, I've used this somewhere, but it is not really suitable for my program, because when notifications are deleted, the user sees a message that he has read everything, so sometimes size just changes from 1 to 1